wire-ios/Wire-iOS UnitTests/NetworkConditionHelperTests.swift

```diff
@@ -1,53 +0,0 @@
-// Wire
-// Copyright (C) 2019 Wire Swiss GmbH
-//
-// This program is free software: you can redistribute it and/or modify
-// it under the terms of the GNU General Public License as published by
-// the Free Software Foundation, either version 3 of the License, or
-// (at your option) any later version.
-//
-// This program is distributed in the hope that it will be useful,
-// but WITHOUT ANY WARRANTY; without even the implied warranty of
-// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
-// GNU General Public License for more details.
-//
-// You should have received a copy of the GNU General Public License
-// along with this program. If not, see http://www.gnu.org/licenses/.
-//
-
-import XCTest
-@testable import Wire
-import CoreTelephony
-
-final class NetworkConditionHelperTests: XCTestCase {
-
-    var sut: NetworkConditionHelper!
-
-    override func setUp() {
-        super.setUp()
-        sut = NetworkConditionHelper()
-    }
-
-    override func tearDown() {
-        sut = nil
-        super.tearDown()
-    }
-
-    // NOTE: this test can fail if your local network conditions are bad/offline?!
-    func testThatSharedInstanceReturnQualityTypeWifi() throws {
-        throw XCTSkip("disabled with the Xcode 15.3 update temporarily!")
-
-        SessionManager.shared?.markNetworkSessionsAsReady(true)
-        XCTAssertEqual(NetworkConditionHelper.shared.qualityType(), .typeWifi)
-    }
```

Comment:
The correct replacement is `testQualityType_givenConnectionAndIsNotMobile_thenIsWifi`